pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/models/MergeResult.scala

```diff
@@ -0,0 +1,14 @@
+package uk.ac.wellcome.platform.merger.models
+
+import uk.ac.wellcome.models.work.internal.{
+  MergedImage,
+  UnidentifiedWork,
+  Unminted
+}
+
+/*
+ * MergeResult holds the resultant target after all fields have been merged,
+ * and the images that were created in the process
+ */
+case class MergeResult(target: UnidentifiedWork,
```

Comment:
Could this be more explicit and be called `mergedTarget` or `result`? (`target` to me meaning it's still... well the `target`.)